### ISSUE TITLE
Screwbots incomplete white screwbot cue

### ISSUE THREAD

**Issue by Strawzzboy64** [chrislo27/RhythmHeavenRemixEditor2#78]
### Prerequisites
* [x] This issue specifically has something to do with RHRE2
* [x] I have attempted to [look for similar issues](https://github.com/chrislo27/RhythmHeavenRemixEditor/issues?utf8=%E2%9C%93&q=is%3Aissue)
already

### Description
The white screwbot cue is missing crane noises which are in the editor.

### Steps to Reproduce the Problem
1. Place White Screwbot cue
2. ???
3. Profit

**Expected behavior:** [insert what was supposed to happen here]
There should be the two crane noises that accompany the two "whistle" sfx at the beginning of the white screwbot cue

**Actual behavior:** [insert what actually happened here]
The crane noises are missing. The cue is otherwise perfect.

**Stacktrace/error log:** [attach the log file from `logs/log_YYYY-MM-DD_HH-MM-SS.txt here]

**Version affected:** [insert the version of RHRE2 here]

### Additional Information
Basically, the two screwbot cues are exactly the same. However, the white screwbot cue adds the whistle noise as the crane picks up the screwbot so you know that it's a white screwbot and not a black screwbot. That's the audio indication.

**Comment by chrislo27** [comment]
Invalid without a version indicated.
